ebos/equil/equilibrationhelpers.hh

```diff
@@ -612,6 +612,8 @@ private:
  */
 class EquilReg
 {
+    using TabulatedFunction = typename Opm::Tabulated1DFunction<double>;
```

Comment:
You don't need `typename` here.  There is nothing in the right-hand-side expression that depends on a template parameter.